sklearn/pipeline.py

```diff
@@ -184,6 +184,26 @@ def predict(self, X):
         return self.steps[-1][-1].predict(Xt)
 
     @if_delegate_has_method(delegate='_final_estimator')
+    def fit_predict(self, X, y=None, **fit_params):
+        """Exposes fit_predict on pipelines.
```

Comment:
Not a good one-line description.
Maybe ``Apply fit_transform on all but the last step, apply fit_predict on the last`` or ``Fit pipeline and predict target``? Not sure, but the current one doesn't really say what the method does.